Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp

```diff
@@ -289,6 +309,8 @@ void JitArm64::addx(UGeckoInstruction inst)
 		if (inst.Rc)
 			ComputeRC(gpr.R(d), 0);
 	}
+
+}
```

Comment:
This enclosing bracket needs to be before the last else. Your indention is incorrect, so it is where the misreading is happening.